src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -1718,7 +1720,7 @@ resource pipeline_ToIngestion 'Microsoft.DataFactory/factories/pipelines@2018-06
         typeProperties: {
           variableName: 'date'
           value: {
-            value: '@{substring(split(split(pipeline().parameters.blobPath, \'/\')[sub(length(split(pipeline().parameters.blobPath, \'/\')), 3)], \'-\')[0], 0, 4)}/@{substring(split(split(pipeline().parameters.blobPath, \'/\')[sub(length(split(pipeline().parameters.blobPath, \'/\')), 3)], \'-\')[0], 4, 2)}'
```

Comment:
FYI - Looks like the branch doesn't have the latest. I merged the latest in so this will have a conflict. I introduced a few variables to deal with the folder path changing with the new API version. The exports team has talked about reverting that to bring back the other folder, so I made it handle both cases.